README.md

```diff
@@ -32,13 +32,13 @@ Find the [latest release](https://github.com/TranscribersOfReddit/TranscribersOf
 $ git clone https://github.com/TranscribersOfReddit/TranscribersOfReddit.git tor
 $ cd tor/
 $ git checkout v3.0.1
-$ pip install .
+$ pip install --process-dependency-links .
 ```
 
 OR
 
 ```
-$ pip install 'git+https://github.com/TranscribersOfReddit/TranscribersOfReddit.git@v3.0.1#egg=tor'
+$ pip install --process-dependency-links 'git+https://github.com/TranscribersOfReddit/TranscribersOfReddit.git@v3.0.1#egg=tor'
```

Comment:
Let's set this to not use a version number -- at least for now, we can reasonably assume that we want to use the latest version of tor_core.

Reply:
Of tor_core? That depends on the setup.py since this has it as a dependency. From context I assume that was a typo and you meant tor. Will change. :+1:

Reply:
Yeah, that. o7